<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EstraiCatastoDialogBase</class>
 <widget class="QDialog" name="EstraiCatastoDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Estrai Catasto</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_input">
     <property name="title">
      <string>Input GeoJSON</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="listWidget_inputs"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_add_input">
        <property name="text">
         <string>Aggiungi GeoJSON</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_remove_input">
        <property name="text">
         <string>Rimuovi selezionato</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_output">
     <property name="title">
      <string>Output GPKG</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_output">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_browse_output">
        <property name="text">
         <string>Sfoglia...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Applica stile:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBox_mappa_continua">
        <property name="text">
         <string>Mappa continua</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_stile">
        <item>
         <property name="text">
          <string>nessuno</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>stile wms</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>stile cad</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_run">
     <property name="text">
      <string>Esegui</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_log">
     <property name="text">
      <string>Log</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit_log"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>EstraiCatastoDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>EstraiCatastoDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
